makeservices/easywp

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Your commit message should be in a comment below this line.

Reply:
For good measure, you should also warn that this script only works on a fresh account, i.e. not one that has an existing website or database. Or you can bail out or show a prompt.